### ISSUE TITLE
phpcbf  closes Control Structure on the same line

### ISSUE THREAD

**Issue by szepeviktor** [WordPress-Coding-Standards/WordPress-Coding-Standards#672]
`phpcbf --standard=WordPress-Core cs.php` corrects

```php
if ( nocontrolstructure() )
    return false;
```
as
```php
if ( nocontrolstructure() ) {
    return false; }
```

But `phpcbf --standard=PSR-2 cs.php` works OK
```php
if (nocontrolstructure()) {
    return false;
}
```
Please advise.

**Comment by szepeviktor** [comment]
`phpcbf --standard=PSR2 --sniffs=Generic.ControlStructures.InlineControlStructure,Squiz.WhiteSpace.ScopeClosingBrace cs.php` is a work-around.

**Comment by jrfnl** [comment]
I've tested this (on Windows) and cannot reproduce the issue.
Suggest: close.

Tested with:
PHPCS `master` and `2.7.0` branch
WPCS: `develop` and `0.10.0` branch

Test procedure used:
Test file `test672.inc`:
```php
<?php

if ( nocontrolstructure() )
	return false;

```

Run `phpcbf -p -s -v ./test672.inc --standard=WordPress --suffix=.fixed`

Resulting file `test672.inc.fixed`
```php
<?php

if ( nocontrolstructure() ) {
	return false;
}

```

**Comment by szepeviktor** [comment]
I believe you.

**Comment by szepeviktor** [comment]
BTW Could you add a test for it?

**Comment by jrfnl** [comment]
> Could you add a test for it?

@szepeviktor Unfortunately no as the sniff being used is an upstream one as sniff libraries can/should only test their own sniffs. All the same, the upstream one contains plenty of tests:
https://github.com/squizlabs/PHP_CodeSniffer/blob/master/CodeSniffer/Standards/Generic/Tests/ControlStructures/InlineControlStructureUnitTest.inc
https://github.com/squizlabs/PHP_CodeSniffer/blob/master/CodeSniffer/Standards/Generic/Tests/ControlStructures/InlineControlStructureUnitTest.inc.fixed

**Comment by szepeviktor** [comment]
!!!
```php
if ($something) echo 'hello';
// becomes
if ($something) { echo 'hello';
}

```

**Comment by GaryJones** [comment]
> !!!

Each sniff's test only checks one aspect - so here, that inline code gets wrapped in `{` and `}`. Other aspects of phpbf would sort the formatting

**Comment by jrfnl** [comment]
Yes, it does. There's another sniff which is involved in that and the part to force a new line has been disabled:
```xml
		<rule ref="Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace">
			<severity>0</severity>
		</rule>
```

I presume that was done to allow code like `if ($something) { echo 'hello'; }` to pass.

**Comment by szepeviktor** [comment]
Thanks. I hope there will be no reopen.
